src/Microsoft.Health.Fhir.Shared.Web/Startup.cs

```diff
@@ -4,30 +4,80 @@
 // -------------------------------------------------------------------------------------------------
 
 using System;
+using Hl7.Fhir.Model;
+using HotChocolate.AspNetCore;
+using MediatR;
 using Microsoft.ApplicationInsights.Extensibility;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.HttpOverrides;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Api.Features.GraphQl.DataLoader;
 using Microsoft.Health.Fhir.Azure;
+using Microsoft.Health.Fhir.Shared.Api.Features.GraphQl;
 
 namespace Microsoft.Health.Fhir.Web
 {
     public class Startup
     {
+        private const string Path = "../Microsoft.Health.Fhir.Core/Data/GraphQl/";
+
         public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
         }
 
         public IConfiguration Configuration { get; }
 
-        // This method gets called by the runtime. Use this method to add services to the container.
         public virtual void ConfigureServices(IServiceCollection services)
         {
             services.AddDevelopmentIdentityProvider(Configuration);
 
+            services
+                .AddRouting()
+
+                // Adding the GraphQL server core service
+                .AddGraphQLServer()
+
+                // Adding our scheme
+                .AddDocumentFromFile(Path + "patient.graphql")
```

Comment:
>Path + "patient.graphql" [](http://example.com/codeflow?start=37&length=24)

You don't have to use DocumentFromFile.
Files are convinent, but hard to pass around.

There is `AddDocumentFromString` function which you can use.

Also instead of `Path + "patient.graphql"`
you can have static `GraphQlLoader` class inside Microsoft.Health.Fhir.Core project
which would have `GetDefinition` method and return string (or explode) for specific type.


`.AddDocument(GraphQlLoader.GetDefinition("patient"))`
something like that. And inside that loader since it's part of Core project you will have access to internal paths. 
So it would avoid adding dependency on path location.

Reply:
To avoid using files, I can try using `AddDocumentFromString` exporting a constant that would have the schema (?)

Reply:
Done! Thank you Ivan